locales/ro/validation.php

```diff
@@ -104,30 +104,30 @@
     'prohibited'           => 'Câmpul :attribute este interzis.',
     'prohibited_if'        => 'Câmpul :attribute este interzis atunci când :other este :value.',
     'prohibited_unless'    => 'Câmpul :attribute este interzis, cu excepția cazului în care :other este în :values.',
-    'prohibits'            => 'The :attribute field prohibits :other from being present.',
+    'prohibits'            => 'Câmpul :attribute nu permite ca :other să fie prezent.',
     'regex'                => 'Câmpul :attribute nu are un format valid.',
     'required'             => 'Câmpul :attribute este obligatoriu.',
-    'required_array_keys'  => 'The :attribute field must contain entries for: :values.',
+    'required_array_keys'  => 'Câmpul :attribute trebuie să conțină valori pentru: :values.',
     'required_if'          => 'Câmpul :attribute este necesar când :other este :value.',
     'required_unless'      => 'Câmpul :attribute este necesar, cu excepția cazului :other este in :values.',
     'required_with'        => 'Câmpul :attribute este necesar când există :values.',
     'required_with_all'    => 'Câmpul :attribute este necesar când există :values.',
     'required_without'     => 'Câmpul :attribute este necesar când nu există :values.',
-    'required_without_all' => 'Câmpul :attribute este necesar când niciunul(una) dintre :values nu există.',
-    'same'                 => 'Câmpul :attribute și :other trebuie să fie identice.',
+    'required_without_all' => 'Câmpul :attribute este necesar când niciuna dintre valorile :values nu există.',
+    'same'                 => 'Câmpurile :attribute și :other trebuie să fie identice.',
```

Comment:
Other keys use the `Câmpul` word. Is it correct that the word `Câmpurile` is used in this variant?

Reply:
Yes. "Câmpul" means "Field" and "Câmpurile" mean "Fields".
On the _same_ rule, it is pluralised, meaning "Fields :attribute and :other have to be identical."